packages/g/glfw/xmake.lua

```diff
@@ -20,6 +20,7 @@ package("glfw")
 
     add_deps("cmake")
 
+    add_deps("glu", "opengl", {optional = true})
```

Comment:
GLU is kinda old, are you sure GLFW requires it?

Reply:
I have removed it.